docs/src/add-services/redis.md

```diff
@@ -31,61 +27,59 @@ Otherwise, they're identical.
-{{< /note >}}
+Note that versions 3.0 and higher support up to 64 different databases per instance of the service,
+while Redis 2.8 only supports a single database.
 
 ## Service types
 
-There are two types of the Redis service depending on what you want to do with it.
+Depending on your needs,
+you can set up Redis as [ephemeral](#ephemeral-redis) or [persistent](#persistent-redis).
 
 ### Ephemeral Redis
 
-The ephemeral Redis service is configured to serve as a cache; its storage isn't persistent.
+By default, Redis is an ephemeral service that serves as a non-persistent cache.
+Ephemeral Redis stores data only in memory and requires no disk space.
+When the service reaches its memory limit, it triggers a cache cleanup.
+To customize those cache cleanups, set up an [eviction policy](#eviction-policy).
 
-Data in an ephemeral Redis instance is stored only in memory and thus requires no disk space.
-When the service hits its memory limit,
-it automatically evicts cache items according to the [configured eviction rule](#eviction-policy) to make room for new ones.
+When you use ephemeral Redis,
+make sure the cache is cleaned up regularly.
+To trigger a cache cleanup every time your app starts,
+configure a `start` [web command](../create-apps/app-reference.md#web-commands) similar to:
 
-Your app must not treat ephemeral Redis as permanent.
-Instead, the cache needs to be regenerated as necessary.
-For example, if a container is moved for a reason such as region maintenance,
-the `deploy` and `post_deploy` hooks don't run and an app that treats the cache as permanent shows errors.
-The cache should be cleared each time the app is restarted,
-in the `start` key in [your web configuration](../create-apps/app-reference.md#web-commands).
+```yaml {location=".platform.app.yaml"}
+web:
+    commands:
+        start: 'redis-cli -h redis.internal flushall'
+```
 
-If your app needs to treat the cache as permanent, use [persistent Redis](#persistent-redis),
-which saves data to its volume even when the container is shut down.
+Note that when you use ephemeral Redis,
+data can be lost when a container is moved or shut down.
+To prevent this, use the [persistent Redis](#persistent-redis) configuration 
+that sets up a Redis cache with persistent storage.
 
 ### Persistent Redis
 
-The `redis-persistent` service type is configured for persistent storage.
-That makes it a good choice for fast application-level key-value storage.
-
-{{< note >}}
-
-You can't switch a service from persistent to ephemeral.
-To switch between modes, use a different service with a different name.
+By default, Redis is an [ephemeral](#ephemeral-redis) service that stores data in memory.
+This allows for fast data retrieval,
+but also means data can be lost when a container is moved or shut down.
 
-{{< /note >}}
+To solve this issue, configure your Redis service as persistent.
```

Comment:
:pick: I don't know, but I feel like `solve` is too strong or something like that. :thinking: 
```suggestion
To resolve this issue, configure your Redis service as persistent.
```